core/classes/Util.php

```diff
@@ -594,4 +541,86 @@ public static function replaceAnchorsWithText($data) {
 		}, $data);
 		return $data;
 	}
+
+	public static function getSetting(DB $db, $setting, $fallback = null) {
```

Comment:
Can we remove the DB types from here + lines 551 and 557?

Reply:
As per our conversation on Discord, leaving this.